catalog/core/catalog-core-api/src/main/java/ddf/catalog/transform/MetacardEnricher.java

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
Yeah, this is basically what I was looking for. I am not sure this is the right package to put it in or the right name to use, but can live with it for now.  I am guessing others would have opinions.  I tried to think if this concept could align with a postIngestPlugin but I don't think that is the right answer, I prefer this.

May want to put something in here that mentioned this is beta (preliminary? experimental?  what's the right word?) and subject to change